dex_chain.json

```diff
@@ -46,5 +46,9 @@
     {
         "id": "babyswap",
         "chain": "BSC"
+     },
+    {
+        "id": "uniswap_v2",
+        "chain": "Ethereum"
```

Comment:
Make this a python file which exposes a constant
```
DEX_CHAIN = {
    "uniswap_v2": "Ethereum",
    ...
}
```
(sort the entries)

Reply:
Changed

Reply:
And then delete this file?